x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_flyout.tsx

```diff
@@ -88,6 +88,18 @@ export const CreateTrustedAppFlyout = memo<CreateTrustedAppFlyoutProps>(
         policies.options.forEach((policy) => {
           errorMessage = errorMessage?.replace(policy.id, policy.name);
         });
+      } else if (
+        creationErrors &&
+        creationErrors.attributes &&
+        creationErrors.attributes.type === 'EndpointLicenseError'
```

Comment:
❤️

Reply:
But why the `attributes`? you should use the existing `meta` property that `EndpointError` allows you to set. see my suggestion below

Reply:
![Screenshot 2021-09-29 at 17 35 11](https://user-images.githubusercontent.com/15727784/135301546-01c393fc-4189-4ec1-a872-3afb8cf7c2ca.png)